catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceMetacardImplTest.java

```diff
@@ -40,16 +51,107 @@ public void testWorkspaceMetacardFrom() {
     assertThat(workspace.getId(), is("0"));
   }
 
+  @Test
+  public void testGetWorkspaceQueries() {
+    WorkspaceMetacardImpl workspaceMetacard = new WorkspaceMetacardImpl();
+    workspaceMetacard.setQueries(Arrays.asList("queryId1", "queryId2"));
+
+    List<String> queries = workspaceMetacard.getQueries();
+
+    assertThat(queries, hasSize(2));
+    assertThat(queries.get(0), is("queryId1"));
+    assertThat(queries.get(1), is("queryId2"));
+  }
+
+  @Test
+  public void testGetEmptyWorkspaceQueries() {
+    WorkspaceMetacardImpl workspaceMetacard = new WorkspaceMetacardImpl();
+    List<String> queries = workspaceMetacard.getQueries();
+
+    assertThat(queries, hasSize(0));
+  }
+
+  @Test
+  public void testAddWorkspaceQuery() {
+    WorkspaceMetacardImpl workspaceMetacard = new WorkspaceMetacardImpl();
+    workspaceMetacard.addQueryAssociation("queryId");
+
+    List<String> queries = workspaceMetacard.getQueries();
+
+    assertThat(queries, hasSize(1));
+    assertThat(queries, contains("queryId"));
+  }
+
+  @Test
+  public void testRemoveWorkspaceQuery() {
+    WorkspaceMetacardImpl workspaceMetacard = new WorkspaceMetacardImpl();
+    workspaceMetacard.addQueryAssociation("queryId1");
+    workspaceMetacard.addQueryAssociation("queryId2");
+
+    workspaceMetacard.removeQueryAssociation("queryId1");
+
+    List<String> queries = workspaceMetacard.getQueries();
+
+    assertThat(queries, hasSize(1));
+    assertThat(queries, contains("queryId2"));
+  }
+
+  @Test
+  public void testGetWorkspaceMetacardFromMap() {
+    final ImmutableMap<String, Serializable> metacardAttributes =
+        new ImmutableMap.Builder<String, Serializable>()
+            .put(Core.ID, "workspaceId")
+            .put(Core.METACARD_TAGS, "workspace")
+            .put(Associations.RELATED, "queryId")
+            .put(Core.TITLE, "title")
+            .put(Core.METACARD_OWNER, "admin")
+            .build();
+
+    WorkspaceMetacardImpl workspaceMetacard = workspace.from(metacardAttributes);
+
+    assertAttributeValueIs(workspaceMetacard, Core.ID, "workspaceId");
+    assertAttributeValueIs(workspaceMetacard, Core.METACARD_TAGS, "workspace");
+    assertAttributeValueIs(workspaceMetacard, Associations.RELATED, "queryId");
+    assertAttributeValueIs(workspaceMetacard, Core.TITLE, "title");
+    assertAttributeValueIs(workspaceMetacard, Core.METACARD_OWNER, "admin");
+  }
+
+  @Test
+  public void testIsWorkspaceMetacard() {
+    MetacardImpl metacard = new MetacardImpl();
+    metacard.setTags(ImmutableSet.of("workspace"));
+
+    boolean isWorkspace = WorkspaceMetacardImpl.isWorkspaceMetacard(metacard);
+
+    assertTrue(isWorkspace);
```

Comment:
When using `assertTrue()/assertFalse()`, we should use the variant that takes a failure message so we can provide more information when the test fails (otherwise the printed failure message will just say `"expected true but was false"`).